sox.features.js

```diff
@@ -37,11 +43,14 @@
     markEmployees: function () {
       // Description: Adds the Stack Exchange logo next to users that *ARE* Stack Exchange employees
 
-      const $icon = '<svg style="color:' + $('.mod-flair').css('color') + '" aria-hidden="true" class="sox-markEmployees-logo sox-sprite svg-icon iconStackExchange" width="18" height="18" viewBox="0 0 18 18"><path d="M15 1H3a2 2 0 0 0-2 2v2h16V3a2 2 0 0 0-2-2zM1 13c0 1.1.9 2 2 2h8v3l3-3h1a2 2 0 0 0 2-2v-2H1v2zm16-7H1v4h16V6z"></path></svg>';
+      const icon = sox.sprites.getSvg('se_logo');
+      icon.classList.add('iconStackExchange');
```

Comment:
I think this can just be `icon.classList.add('iconStackExchange', 'svg-icon', ...);`